src/main/webapp/WEB-INF/view/activity.jsp

```diff
@@ -54,18 +60,24 @@ MessageStore messageStore = (MessageStore) request.getAttribute("messageStore");
 <div id="feed">
   <ul>
     <%
-      /* Get the titles of the conversations. */
+       /* Used to format the time since the getCreationTime() method returns an instance, not a string. */
+       DateTimeFormatter formatter =DateTimeFormatter.ofLocalizedDateTime( FormatStyle.SHORT ).withLocale( Locale.US ).withZone( ZoneId.systemDefault() );
+       
+       /* Get the titles of the conversations. */
        for (Conversation conversation : conversations) {
          String title = conversation.getTitle();
     %>
-       <li><strong><%= title %></strong></li>
        <%
-            /* Get the messages of the conversations. */
+         /* Get the messages of the conversations. */
          List<Message> messages = messageStore.getMessagesInConversation(conversation.getId());
          for (Message message : messages) {
        %>
-           
-           <li><%= message.getContent() %></li>  
+           <%
+             String output = formatter.format( message.getCreationTime() );
```

Comment:
Rather than 'output', 'formatttedCreationTime' would be much better name for this variable.